datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/JSR310DateTimeDeserializerBase.java

```diff
@@ -92,6 +110,12 @@ protected JSR310DateTimeDeserializerBase(JSR310DateTimeDeserializerBase<T> base,
                     deser = deser.withLeniency(leniency);
                 }
             }
+            //if (format.hasShape()) {
+                JsonFormat.Shape shape = format.getShape();
```

Comment:
Probably want to check existing `_shape` just because `createContextual()` might get called multiple times (should not be, but should be define idempotent just in case).